iotdb-core/datanode/src/assembly/resources/sbin/remove-datanode.bat

```diff
@@ -125,7 +125,7 @@ goto :eof
 
 rem echo CLASSPATH: %CLASSPATH%
 
-"%JAVA_HOME%\bin\java" %JAVA_OPTS% %IOTDB_HEAP_OPTS% -cp %CLASSPATH% %IOTDB_JMX_OPTS% %MAIN_CLASS% %CONF_PARAMS%
+"%JAVA_HOME%\bin\java" %JAVA_OPTS% %IOTDB_HEAP_OPTS% -Xms2g -Xmx2g -cp %CLASSPATH% %IOTDB_JMX_OPTS% %MAIN_CLASS% %CONF_PARAMS%
```

Comment:
Can we set two new variables in this configuration file to indicate out-of-heap and in-heap memory? In case the 2g memory is not enough in some scenarios, users can further reduce the memory usage manually